### DISCUSSION TITLE
Feature Request: In and Out markers in timeline

### DISCUSSION THREAD

**Discussion by MishiGeneration** [p2or/blender-loom#97] [Ideas]
Hello, first of all, love the addon!  Makes it much more efficient to render animations while keeping them in sync with audio, thank you for making it so accessible.

Just a humble feature request from me though:  In and Out markers in the timeline, dopesheet, etc. would save sooo much time.  I will clarify exactly what I mean by "in and out markers"
A new button in the marker menu of the timeline that would allow users to "mark start of image sequence" and "mark end of image sequence"  Instead of having to write down the frame ranges manually, it would allow users to simply scroll through their timeline and (ideally with a hot key) mark the in and out of the ranges there. 

I make animations in the 5 digit frame number range, and having to write down all the in's and out's for the render sequence ranges gets a bit tedious.  
I know loom "adds three new operators to the Marker," and I admit I dont quite understand what they do.  But I don't think they are do what I'm describing, please correct me if I'm wrong.
Thank you for your time and consideration!

**Comment by p2or** [comment]
Hi @MishiGeneration,

thanks for your kind words and your request. Sounds like a great UI improvement! I could even imagine that the ability to create multiple ranges (e.g. in different colors) would be even more helpful. Unfortunately I have a project to work on at the moment, but I will implement an initial prototype in 3-4 weeks and let you know. Cool?

Cheers,
Christian

**Reply by MishiGeneration** [reply]
Yay thank you!  Yes multiple ranges in different colors would be very helpful.  Best of luck on your project, I'll be looking forward to the prototype!
